src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

```diff
@@ -0,0 +1,64 @@
+package app.revanced.patches.youtube.layout.spoofappversion.bytecode.patch
+
+import app.revanced.extensions.toErrorResult
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.shared.settings.preference.impl.StringResource
+import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
+import app.revanced.patches.youtube.layout.spoofappversion.annotations.SpoofAppVersionCompatibility
+import app.revanced.patches.youtube.layout.spoofappversion.bytecode.fingerprints.SpoofAppVersionFingerprint
+import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
+import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch
+@DependsOn([IntegrationsPatch::class, SettingsPatch::class])
+@Name("spoof-app-version")
+@Description("Makes YouTube think the app is an older version.  This gives the old UI layout, but may cause unknown side effects.")
+@SpoofAppVersionCompatibility
+@Version("0.0.1")
+class SpoofAppVersionPatch : BytecodePatch(
+    listOf(
+        SpoofAppVersionFingerprint
+    )
+) {
+    companion object {
+        const val INTEGRATIONS_CLASS_DESCRIPTOR = "Lapp/revanced/integrations/patches/SpoofAppVersionPatch"
+    }
+
+    override fun execute(context: BytecodeContext): PatchResult {
+        SettingsPatch.PreferenceScreen.LAYOUT.addPreferences(
+            SwitchPreference(
+                "revanced_spoof_app_version",
+                StringResource("revanced_spoof_app_version_title", "Spoof app version (Show old UI)"),
+                false,
+                StringResource("revanced_spoof_app_version_summary_on", "Version spoofed to 17.30.34.  If switched off, the old UI layout may remain until user logs out or the app data is cleared"),
```

Comment:
Through the comment, the old UI is implicated, allowing for the title to be short and clean.

```suggestion
                StringResource("revanced_spoof_app_version_title", "Spoof app version"),
                false,
                StringResource("revanced_spoof_app_version_summary_on", "Version spoofed to 17.30.34. If switched off, the old UI layout may remain until logging out or clearing app data"),
```